pkg/agent/externalnode/external_node_controller_windows.go

```diff
@@ -14,8 +14,43 @@
 
 package externalnode
 
-import "antrea.io/antrea/pkg/agent/config"
+import (
+	"k8s.io/klog/v2"
 
+	"antrea.io/antrea/pkg/agent/config"
+	"antrea.io/antrea/pkg/agent/util"
+	"antrea.io/antrea/pkg/signals"
+)
+
+// moveIFConfigurations returns nil for single interface case, as it relies
+// on Windows New-VMSwitch command to create a host network adapter and copy
+// the uplink adapter configurations to host adapter
+// TODO: Implement the function to handle multiple interface case.
+// It should perform the following operations
+// Enable the host interface after it is created by OVS.
+// Update the host interface MAC address with uplink's.
+// Copy the uplink interface's IP to the host interface.
+// Copy the uplink interface's Route to the host interface.
 func (c *ExternalNodeController) moveIFConfigurations(adapterConfig *config.AdapterNetConfig, from string, to string) error {
 	return nil
 }
+
+// TODO: Handle multiple interface case
+// For multiple interface case, should remove VMSwitch only
+// when last interface is deleted from the ExternalNode
+func (c *ExternalNodeController) removeExternalNodeConfig() error {
+	if ovsErr := c.ovsBridgeClient.Delete(); ovsErr != nil {
+		klog.ErrorS(ovsErr, "Failed to delete OVS bridge")
+	}
+
+	ok, err := util.RemoveVMSwitch()
+	if err != nil {
+		return err
+	}
+	if !ok {
+		klog.ErrorS(err, "Could not delete VM Switch")
+	}
+	// stop process
```

Comment:
Please add a few more words to explain this, or you can do that before removeExternalNodeConfig().

Reply:
Sure